DxDiagram improve Expr descriptions (#620)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxDiagram/1 Configuration/edges/lockedExpr.md

The current edge's data object.

---
A locked item can not be moved, changed, or delete. Users can use the [context menu](/Documentation/Guide/Widgets/Diagram/Diagram_Tools/) to lock and unlock an item.

> The specified field or expression must return `true` or `false` value.

Reviewer: ```suggestion
The specified field or expression should return `true` or `false`.
```
